app/assets/stylesheets/theme-waters.scss

```diff
@@ -0,0 +1,265 @@
+      color-adjust: exact;
+    }
+    .logo-r {
+      content: url(/assets/waters/students.jpg);
+    }
+  }
+
+  #main {
+
+    .activity-mod {
+      margin-bottom: 10px;
+
+      .activity-nav-mod + .site-width {
+        background: $col-white;
+        padding: 20px;
+        width: 960px;
+      }
+    } // end .activity-mod
+
+    .activity-footer-mod {
+      margin-bottom: 10px;
+    }
+  } // end #main
+
+  footer {
+    background: $col-lightgray;
+
+    .footer-txt { text-shadow: none; }
+  }
+} // end #container
+
+.activity-nav-mod {
+  margin: 0;
+
+  #nav-activity-menu-container {
+    margin: 0 auto;
+    position: relative;
+    width: 960px;
+  } // end #nav-activity-menu-container
+
+  table {
+    background: $col-white;
+    border-right: solid 20px $col-white;
+    border-left: solid 20px $col-white;
+    color: $col-darkgray;
+
+    td:last-child {
+      max-width: 110px;
+      text-align: right;
+      width: 110px;
+
+      &.ui-block-2 {
+        width: auto;
+      }
+    }
+  }
+
+  .activity-nav-title {
+    color: $col-darkgray-50;
+    font-weight: normal;
+  }
+  .profile-nav {
+    background: $col-white;
+
+    .profile-dropdown { right: 20px; }
+  }
+
+  .activity-nav .prev_and_next .prev, .activity-nav .prev_and_next .next {
+    background: #dbf1f5;
+    border: none;
+    color: $col-lightblue;
+    height: 32px;
+    line-height: 1.9;
+    -moz-border-radius: 0;
+    -webkit-border-radius: 0;
+    border-radius: 0;
+  }
+
+  .pagination {
+    margin-right: 0;
+
+    .pagination-item .pagination-link {
+      border: none;
+      color: $col-lightblue;
+      padding: 2px 6px 3px;
+      text-shadow: none;
+      -moz-border-radius: 0;
+      -webkit-border-radius: 0;
+      border-radius: 0;
+
+      &:hover { color: $col-white; }
+      &.disabled { opacity: .5; }
+      &.selected { color: $col-white; }
+    }
+    .pagination-item:last-child {
+      margin-right: 0;
+    }
+  } // end .pagination
+
+  &.fixed {
+    background-color: transparent;
+    position: relative;
+    top: auto;
+    -moz-box-shadow: none;
+    -webkit-box-shadow: none;
+    box-shadow: none;
+    padding: 0;
+    z-index: 1;
+  }
+} // end .activity-nav-mod
+
+#nav-activity-menu {
+  border: none;
+  -moz-border-radius: 0;
+  -webkit-border-radius: 0;
+  border-radius: 0;
+  color: #313131;
+  font-size: 12px;
+  margin-top: 0;
+  position: relative;
+
+  ul {
+    padding: 10px 0 5px 20px;
+    text-align: left;
+
+    li {
+      border-bottom: none;
+      display: inline-block;
+      padding: 0;
+
+      &.am-print { display: inline-block; }
+      &.am-report, &.am-close { display: none; }
+
+      a {
+        color: #007c8b;
+        display: block;
+        opacity: .7;
+        padding: 0;
+
+        &:hover {
+          opacity: 1;
+        }
+        i.fa { display: none; }
+        span.text { display: inline; }
+      }
+    }
+  }
+} // end #nav-activity-menu
+
+.precipitating-change-footer {
```

Comment:
Is this correct that there is a precipitating-change footer element?
Is that because the activity has a project setting of precipitating-change?

Reply:
Thanks for catching that. Those should've been updated from 'precipitating-change' to 'waters-'. I've made that change.